chai 2024.02.1 fails to build against raja 2024.02.2: `error: 'loop_exec' is not a member of 'RAJA'`

```
[ 95%] Built target managed_ptr_unit_tests
/opt/local/var/macports/build/_opt_PPCSnowLeopardPorts_devel_chai/chai/work/CHAI-2024.02.1/tests/integration/raja-chai-launch.cpp: In function 'void cuda_test_Chai_LaunchSimple()':
/opt/local/var/macports/build/_opt_PPCSnowLeopardPorts_devel_chai/chai/work/CHAI-2024.02.1/tests/integration/raja-chai-launch.cpp:51:48: error: 'loop_exec' is not a member of 'RAJA'
   51 |   using LOOP_POLICY_0 = RAJA::LoopPolicy<RAJA::loop_exec>;
      |                                                ^~~~~~~~~
/opt/local/var/macports/build/_opt_PPCSnowLeopardPorts_devel_chai/chai/work/CHAI-2024.02.1/tests/integration/raja-chai-launch.cpp:51:57: error: template argument 1 is invalid
   51 |   using LOOP_POLICY_0 = RAJA::LoopPolicy<RAJA::loop_exec>;
      |                                                         ^
/opt/local/var/macports/build/_opt_PPCSnowLeopardPorts_devel_chai/chai/work/CHAI-2024.02.1/tests/integration/raja-chai-launch.cpp:52:48: error: 'loop_exec' is not a member of 'RAJA'
   52 |   using LOOP_POLICY_1 = RAJA::LoopPolicy<RAJA::loop_exec>;
      |                                                ^~~~~~~~~
/opt/local/var/macports/build/_opt_PPCSnowLeopardPorts_devel_chai/chai/work/CHAI-2024.02.1/tests/integration/raja-chai-launch.cpp:52:57: error: template argument 1 is invalid
   52 |   using LOOP_POLICY_1 = RAJA::LoopPolicy<RAJA::loop_exec>;
      |                                                         ^
/opt/local/var/macports/build/_opt_PPCSnowLeopardPorts_devel_chai/chai/work/CHAI-2024.02.1/tests/integration/raja-chai-launch.cpp:64:57: error: 'loop_exec' is not a member of 'RAJA'
   64 |   using PARALLEL_LOOP_POLICY_0 = RAJA::LoopPolicy<RAJA::loop_exec>;
      |                                                         ^~~~~~~~~
/opt/local/var/macports/build/_opt_PPCSnowLeopardPorts_devel_chai/chai/work/CHAI-2024.02.1/tests/integration/raja-chai-launch.cpp:64:66: error: template argument 1 is invalid
   64 |   using PARALLEL_LOOP_POLICY_0 = RAJA::LoopPolicy<RAJA::loop_exec>;
      |                                                                  ^
/opt/local/var/macports/build/_opt_PPCSnowLeopardPorts_devel_chai/chai/work/CHAI-2024.02.1/tests/integration/raja-chai-launch.cpp: In lambda function:
/opt/local/var/macports/build/_opt_PPCSnowLeopardPorts_devel_chai/chai/work/CHAI-2024.02.1/tests/integration/raja-chai-launch.cpp:83:19: error: 'LOOP_POLICY_1' was not declared in this scope
   83 |        RAJA::loop<LOOP_POLICY_1>(ctx, RAJA::RangeSegment(0, X), [&] (int i) {
      |                   ^~~~~~~~~~~~~
/opt/local/var/macports/build/_opt_PPCSnowLeopardPorts_devel_chai/chai/work/CHAI-2024.02.1/tests/integration/raja-chai-launch.cpp: In lambda function:
/opt/local/var/macports/build/_opt_PPCSnowLeopardPorts_devel_chai/chai/work/CHAI-2024.02.1/tests/integration/raja-chai-launch.cpp:84:23: error: 'LOOP_POLICY_0' was not declared in this scope
   84 |            RAJA::loop<LOOP_POLICY_0>(ctx, RAJA::RangeSegment(0, Y), [&] (int j) {
      |                       ^~~~~~~~~~~~~
/opt/local/var/macports/build/_opt_PPCSnowLeopardPorts_devel_chai/chai/work/CHAI-2024.02.1/tests/integration/raja-chai-launch.cpp:84:37: error: no matching function for call to 'loop<<expression error> >(RAJA::LaunchContext&, RAJA::RangeSegment, cuda_test_Chai_LaunchSimple()::<lambda(RAJA::LaunchContext)>::<lambda(int)>::<lambda(int)>)'
   84 |            RAJA::loop<LOOP_POLICY_0>(ctx, RAJA::RangeSegment(0, Y), [&] (int j) {
      |            ~~~~~~~~~~~~~~~~~~~~~~~~~^~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
   85 |                int index = j * X + i;
      |                ~~~~~~~~~~~~~~~~~~~~~~
   86 |                v1[index] = index;
      |                ~~~~~~~~~~~~~~~~~~    
   87 |              });
      |              ~~                      
In file included from /opt/local/include/RAJA/pattern/launch.hpp:21,
                 from /opt/local/include/RAJA/RAJA.hpp:48,
                 from /opt/local/var/macports/build/_opt_PPCSnowLeopardPorts_devel_chai/chai/work/CHAI-2024.02.1/tests/integration/raja-chai-launch.cpp:19:
/opt/local/include/RAJA/pattern/launch/launch_core.hpp:546:35: note: candidate: 'template<class POLICY_LIST, class CONTEXT, class SEGMENT, class BODY> void RAJA::loop(const CONTEXT&, const SEGMENT&, const BODY&)'
  546 | RAJA_HOST_DEVICE RAJA_INLINE void loop(CONTEXT const &ctx,
      |                                   ^~~~
/opt/local/include/RAJA/pattern/launch/launch_core.hpp:546:35: note:   template argument deduction/substitution failed:
/opt/local/var/macports/build/_opt_PPCSnowLeopardPorts_devel_chai/chai/work/CHAI-2024.02.1/tests/integration/raja-chai-launch.cpp:84:37: error: template argument 1 is invalid
   84 |            RAJA::loop<LOOP_POLICY_0>(ctx, RAJA::RangeSegment(0, Y), [&] (int j) {
      |            ~~~~~~~~~~~~~~~~~~~~~~~~~^~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
   85 |                int index = j * X + i;
      |                ~~~~~~~~~~~~~~~~~~~~~~
   86 |                v1[index] = index;
      |                ~~~~~~~~~~~~~~~~~~    
   87 |              });
      |              ~~                      
/opt/local/var/macports/build/_opt_PPCSnowLeopardPorts_devel_chai/chai/work/CHAI-2024.02.1/tests/integration/raja-chai-launch.cpp: In lambda function:
/opt/local/var/macports/build/_opt_PPCSnowLeopardPorts_devel_chai/chai/work/CHAI-2024.02.1/tests/integration/raja-chai-launch.cpp:83:33: error: no matching function for call to 'loop<<expression error> >(RAJA::LaunchContext&, RAJA::RangeSegment, cuda_test_Chai_LaunchSimple()::<lambda(RAJA::LaunchContext)>::<lambda(int)>)'
   83 |        RAJA::loop<LOOP_POLICY_1>(ctx, RAJA::RangeSegment(0, X), [&] (int i) {
      |        ~~~~~~~~~~~~~~~~~~~~~~~~~^~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
   84 |            RAJA::loop<LOOP_POLICY_0>(ctx, RAJA::RangeSegment(0, Y), [&] (int j) {
      |            ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
   85 |                int index = j * X + i;
      |                ~~~~~~~~~~~~~~~~~~~~~~
   86 |                v1[index] = index;
      |                ~~~~~~~~~~~~~~~~~~
   87 |              });
      |              ~~~                 
   88 |          });
      |          ~~                      
/opt/local/include/RAJA/pattern/launch/launch_core.hpp:546:35: note: candidate: 'template<class POLICY_LIST, class CONTEXT, class SEGMENT, class BODY> void RAJA::loop(const CONTEXT&, const SEGMENT&, const BODY&)'
  546 | RAJA_HOST_DEVICE RAJA_INLINE void loop(CONTEXT const &ctx,
      |                                   ^~~~
/opt/local/include/RAJA/pattern/launch/launch_core.hpp:546:35: note:   template argument deduction/substitution failed:
/opt/local/var/macports/build/_opt_PPCSnowLeopardPorts_devel_chai/chai/work/CHAI-2024.02.1/tests/integration/raja-chai-launch.cpp:83:33: error: template argument 1 is invalid
   83 |        RAJA::loop<LOOP_POLICY_1>(ctx, RAJA::RangeSegment(0, X), [&] (int i) {
      |        ~~~~~~~~~~~~~~~~~~~~~~~~~^~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
   84 |            RAJA::loop<LOOP_POLICY_0>(ctx, RAJA::RangeSegment(0, Y), [&] (int j) {
      |            ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
   85 |                int index = j * X + i;
      |                ~~~~~~~~~~~~~~~~~~~~~~
   86 |                v1[index] = index;
      |                ~~~~~~~~~~~~~~~~~~
   87 |              });
      |              ~~~                 
   88 |          });
      |          ~~                      
/opt/local/var/macports/build/_opt_PPCSnowLeopardPorts_devel_chai/chai/work/CHAI-2024.02.1/tests/integration/raja-chai-launch.cpp: In lambda function:
/opt/local/var/macports/build/_opt_PPCSnowLeopardPorts_devel_chai/chai/work/CHAI-2024.02.1/tests/integration/raja-chai-launch.cpp:96:23: error: 'PARALLEL_LOOP_POLICY_0' was not declared in this scope; did you mean 'PARALLEL_LOOP_POLICY_1'?
   96 |            RAJA::loop<PARALLEL_LOOP_POLICY_0>(ctx, RAJA::RangeSegment(0, Y), [&] (int j) {
      |                       ^~~~~~~~~~~~~~~~~~~~~~
      |                       PARALLEL_LOOP_POLICY_1
/opt/local/var/macports/build/_opt_PPCSnowLeopardPorts_devel_chai/chai/work/CHAI-2024.02.1/tests/integration/raja-chai-launch.cpp:96:46: error: no matching function for call to 'loop<<expression error> >(RAJA::LaunchContext&, RAJA::RangeSegment, cuda_test_Chai_LaunchSimple()::<lambda(RAJA::LaunchContext)>::<lambda(int)>::<lambda(int)>)'
   96 |            RAJA::loop<PARALLEL_LOOP_POLICY_0>(ctx, RAJA::RangeSegment(0, Y), [&] (int j) {
      |            ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~^~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
   97 |                int index = j * X + i;
      |                ~~~~~~~~~~~~~~~~~~~~~~         
   98 |                v2[index] = v1[index] * 2.0f;
      |                ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~  
   99 |              });
      |              ~~                               
/opt/local/include/RAJA/pattern/launch/launch_core.hpp:546:35: note: candidate: 'template<class POLICY_LIST, class CONTEXT, class SEGMENT, class BODY> void RAJA::loop(const CONTEXT&, const SEGMENT&, const BODY&)'
  546 | RAJA_HOST_DEVICE RAJA_INLINE void loop(CONTEXT const &ctx,
      |                                   ^~~~
/opt/local/include/RAJA/pattern/launch/launch_core.hpp:546:35: note:   template argument deduction/substitution failed:
/opt/local/var/macports/build/_opt_PPCSnowLeopardPorts_devel_chai/chai/work/CHAI-2024.02.1/tests/integration/raja-chai-launch.cpp:96:46: error: template argument 1 is invalid
   96 |            RAJA::loop<PARALLEL_LOOP_POLICY_0>(ctx, RAJA::RangeSegment(0, Y), [&] (int j) {
      |            ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~^~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
   97 |                int index = j * X + i;
      |                ~~~~~~~~~~~~~~~~~~~~~~         
   98 |                v2[index] = v1[index] * 2.0f;
      |                ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~  
   99 |              });
      |              ~~                               
/opt/local/var/macports/build/_opt_PPCSnowLeopardPorts_devel_chai/chai/work/CHAI-2024.02.1/tests/integration/raja-chai-launch.cpp: In lambda function:
/opt/local/var/macports/build/_opt_PPCSnowLeopardPorts_devel_chai/chai/work/CHAI-2024.02.1/tests/integration/raja-chai-launch.cpp:108:19: error: 'LOOP_POLICY_1' was not declared in this scope
  108 |        RAJA::loop<LOOP_POLICY_1>(ctx, RAJA::RangeSegment(0, X), [&] (int i) {
      |                   ^~~~~~~~~~~~~
/opt/local/var/macports/build/_opt_PPCSnowLeopardPorts_devel_chai/chai/work/CHAI-2024.02.1/tests/integration/raja-chai-launch.cpp: In lambda function:
/opt/local/var/macports/build/_opt_PPCSnowLeopardPorts_devel_chai/chai/work/CHAI-2024.02.1/tests/integration/raja-chai-launch.cpp:109:23: error: 'LOOP_POLICY_0' was not declared in this scope
  109 |            RAJA::loop<LOOP_POLICY_0>(ctx, RAJA::RangeSegment(0, Y), [&] (int j) {
      |                       ^~~~~~~~~~~~~
/opt/local/var/macports/build/_opt_PPCSnowLeopardPorts_devel_chai/chai/work/CHAI-2024.02.1/tests/integration/raja-chai-launch.cpp:109:37: error: no matching function for call to 'loop<<expression error> >(RAJA::LaunchContext&, RAJA::RangeSegment, cuda_test_Chai_LaunchSimple()::<lambda(RAJA::LaunchContext)>::<lambda(int)>::<lambda(int)>)'
  109 |            RAJA::loop<LOOP_POLICY_0>(ctx, RAJA::RangeSegment(0, Y), [&] (int j) {
      |            ~~~~~~~~~~~~~~~~~~~~~~~~~^~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
  110 |                int index = j * X + i;
      |                ~~~~~~~~~~~~~~~~~~~~~~
  111 |                ASSERT_FLOAT_EQ(v1[index], index * 1.0f);
      |                ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
  112 |                ASSERT_FLOAT_EQ(v2[index], index * 2.0f);
      |                ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
  113 |              });
      |              ~~                      
/opt/local/include/RAJA/pattern/launch/launch_core.hpp:546:35: note: candidate: 'template<class POLICY_LIST, class CONTEXT, class SEGMENT, class BODY> void RAJA::loop(const CONTEXT&, const SEGMENT&, const BODY&)'
  546 | RAJA_HOST_DEVICE RAJA_INLINE void loop(CONTEXT const &ctx,
      |                                   ^~~~
/opt/local/include/RAJA/pattern/launch/launch_core.hpp:546:35: note:   template argument deduction/substitution failed:
/opt/local/var/macports/build/_opt_PPCSnowLeopardPorts_devel_chai/chai/work/CHAI-2024.02.1/tests/integration/raja-chai-launch.cpp:109:37: error: template argument 1 is invalid
  109 |            RAJA::loop<LOOP_POLICY_0>(ctx, RAJA::RangeSegment(0, Y), [&] (int j) {
      |            ~~~~~~~~~~~~~~~~~~~~~~~~~^~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
  110 |                int index = j * X + i;
      |                ~~~~~~~~~~~~~~~~~~~~~~
  111 |                ASSERT_FLOAT_EQ(v1[index], index * 1.0f);
      |                ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
  112 |                ASSERT_FLOAT_EQ(v2[index], index * 2.0f);
      |                ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
  113 |              });
      |              ~~                      
/opt/local/var/macports/build/_opt_PPCSnowLeopardPorts_devel_chai/chai/work/CHAI-2024.02.1/tests/integration/raja-chai-launch.cpp: In lambda function:
/opt/local/var/macports/build/_opt_PPCSnowLeopardPorts_devel_chai/chai/work/CHAI-2024.02.1/tests/integration/raja-chai-launch.cpp:108:33: error: no matching function for call to 'loop<<expression error> >(RAJA::LaunchContext&, RAJA::RangeSegment, cuda_test_Chai_LaunchSimple()::<lambda(RAJA::LaunchContext)>::<lambda(int)>)'
  108 |        RAJA::loop<LOOP_POLICY_1>(ctx, RAJA::RangeSegment(0, X), [&] (int i) {
      |        ~~~~~~~~~~~~~~~~~~~~~~~~~^~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
  109 |            RAJA::loop<LOOP_POLICY_0>(ctx, RAJA::RangeSegment(0, Y), [&] (int j) {
      |            ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
  110 |                int index = j * X + i;
      |                ~~~~~~~~~~~~~~~~~~~~~~
  111 |                ASSERT_FLOAT_EQ(v1[index], index * 1.0f);
      |                ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
  112 |                ASSERT_FLOAT_EQ(v2[index], index * 2.0f);
      |                ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
  113 |              });
      |              ~~~                 
  114 |          });
      |          ~~                      
/opt/local/include/RAJA/pattern/launch/launch_core.hpp:546:35: note: candidate: 'template<class POLICY_LIST, class CONTEXT, class SEGMENT, class BODY> void RAJA::loop(const CONTEXT&, const SEGMENT&, const BODY&)'
  546 | RAJA_HOST_DEVICE RAJA_INLINE void loop(CONTEXT const &ctx,
      |                                   ^~~~
/opt/local/include/RAJA/pattern/launch/launch_core.hpp:546:35: note:   template argument deduction/substitution failed:
/opt/local/var/macports/build/_opt_PPCSnowLeopardPorts_devel_chai/chai/work/CHAI-2024.02.1/tests/integration/raja-chai-launch.cpp:108:33: error: template argument 1 is invalid
  108 |        RAJA::loop<LOOP_POLICY_1>(ctx, RAJA::RangeSegment(0, X), [&] (int i) {
      |        ~~~~~~~~~~~~~~~~~~~~~~~~~^~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
  109 |            RAJA::loop<LOOP_POLICY_0>(ctx, RAJA::RangeSegment(0, Y), [&] (int j) {
      |            ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
  110 |                int index = j * X + i;
      |                ~~~~~~~~~~~~~~~~~~~~~~
  111 |                ASSERT_FLOAT_EQ(v1[index], index * 1.0f);
      |                ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
  112 |                ASSERT_FLOAT_EQ(v2[index], index * 2.0f);
      |                ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
  113 |              });
      |              ~~~                 
  114 |          });
      |          ~~                      
/opt/local/var/macports/build/_opt_PPCSnowLeopardPorts_devel_chai/chai/work/CHAI-2024.02.1/tests/integration/raja-chai-launch.cpp: In function 'void cuda_test_Chai_LaunchView()':
/opt/local/var/macports/build/_opt_PPCSnowLeopardPorts_devel_chai/chai/work/CHAI-2024.02.1/tests/integration/raja-chai-launch.cpp:125:48: error: 'loop_exec' is not a member of 'RAJA'
  125 |   using LOOP_POLICY_0 = RAJA::LoopPolicy<RAJA::loop_exec>;
      |                                                ^~~~~~~~~
/opt/local/var/macports/build/_opt_PPCSnowLeopardPorts_devel_chai/chai/work/CHAI-2024.02.1/tests/integration/raja-chai-launch.cpp:125:57: error: template argument 1 is invalid
  125 |   using LOOP_POLICY_0 = RAJA::LoopPolicy<RAJA::loop_exec>;
      |                                                         ^
/opt/local/var/macports/build/_opt_PPCSnowLeopardPorts_devel_chai/chai/work/CHAI-2024.02.1/tests/integration/raja-chai-launch.cpp:126:48: error: 'loop_exec' is not a member of 'RAJA'
  126 |   using LOOP_POLICY_1 = RAJA::LoopPolicy<RAJA::loop_exec>;
      |                                                ^~~~~~~~~
/opt/local/var/macports/build/_opt_PPCSnowLeopardPorts_devel_chai/chai/work/CHAI-2024.02.1/tests/integration/raja-chai-launch.cpp:126:57: error: template argument 1 is invalid
  126 |   using LOOP_POLICY_1 = RAJA::LoopPolicy<RAJA::loop_exec>;
      |                                                         ^
/opt/local/var/macports/build/_opt_PPCSnowLeopardPorts_devel_chai/chai/work/CHAI-2024.02.1/tests/integration/raja-chai-launch.cpp:138:57: error: 'loop_exec' is not a member of 'RAJA'
  138 |   using PARALLEL_LOOP_POLICY_0 = RAJA::LoopPolicy<RAJA::loop_exec>;
      |                                                         ^~~~~~~~~
/opt/local/var/macports/build/_opt_PPCSnowLeopardPorts_devel_chai/chai/work/CHAI-2024.02.1/tests/integration/raja-chai-launch.cpp:138:66: error: template argument 1 is invalid
  138 |   using PARALLEL_LOOP_POLICY_0 = RAJA::LoopPolicy<RAJA::loop_exec>;
      |                                                                  ^
/opt/local/var/macports/build/_opt_PPCSnowLeopardPorts_devel_chai/chai/work/CHAI-2024.02.1/tests/integration/raja-chai-launch.cpp: In lambda function:
/opt/local/var/macports/build/_opt_PPCSnowLeopardPorts_devel_chai/chai/work/CHAI-2024.02.1/tests/integration/raja-chai-launch.cpp:164:19: error: 'LOOP_POLICY_1' was not declared in this scope
  164 |        RAJA::loop<LOOP_POLICY_1>(ctx, RAJA::RangeSegment(0, X), [&] (int i) {
      |                   ^~~~~~~~~~~~~
/opt/local/var/macports/build/_opt_PPCSnowLeopardPorts_devel_chai/chai/work/CHAI-2024.02.1/tests/integration/raja-chai-launch.cpp: In lambda function:
/opt/local/var/macports/build/_opt_PPCSnowLeopardPorts_devel_chai/chai/work/CHAI-2024.02.1/tests/integration/raja-chai-launch.cpp:165:23: error: 'LOOP_POLICY_0' was not declared in this scope
  165 |            RAJA::loop<LOOP_POLICY_0>(ctx, RAJA::RangeSegment(0, Y), [&] (int j) {
      |                       ^~~~~~~~~~~~~
/opt/local/var/macports/build/_opt_PPCSnowLeopardPorts_devel_chai/chai/work/CHAI-2024.02.1/tests/integration/raja-chai-launch.cpp:165:37: error: no matching function for call to 'loop<<expression error> >(RAJA::LaunchContext&, RAJA::RangeSegment, cuda_test_Chai_LaunchView()::<lambda(RAJA::LaunchContext)>::<lambda(int)>::<lambda(int)>)'
  165 |            RAJA::loop<LOOP_POLICY_0>(ctx, RAJA::RangeSegment(0, Y), [&] (int j) {
      |            ~~~~~~~~~~~~~~~~~~~~~~~~~^~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
  166 |                v1(i, j) = (i + (j * X)) * 1.0f;
      |                ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
  167 |              });
      |              ~~                      
/opt/local/include/RAJA/pattern/launch/launch_core.hpp:546:35: note: candidate: 'template<class POLICY_LIST, class CONTEXT, class SEGMENT, class BODY> void RAJA::loop(const CONTEXT&, const SEGMENT&, const BODY&)'
  546 | RAJA_HOST_DEVICE RAJA_INLINE void loop(CONTEXT const &ctx,
      |                                   ^~~~
/opt/local/include/RAJA/pattern/launch/launch_core.hpp:546:35: note:   template argument deduction/substitution failed:
/opt/local/var/macports/build/_opt_PPCSnowLeopardPorts_devel_chai/chai/work/CHAI-2024.02.1/tests/integration/raja-chai-launch.cpp:165:37: error: template argument 1 is invalid
  165 |            RAJA::loop<LOOP_POLICY_0>(ctx, RAJA::RangeSegment(0, Y), [&] (int j) {
      |            ~~~~~~~~~~~~~~~~~~~~~~~~~^~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
  166 |                v1(i, j) = (i + (j * X)) * 1.0f;
      |                ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
  167 |              });
      |              ~~                      
/opt/local/var/macports/build/_opt_PPCSnowLeopardPorts_devel_chai/chai/work/CHAI-2024.02.1/tests/integration/raja-chai-launch.cpp: In lambda function:
/opt/local/var/macports/build/_opt_PPCSnowLeopardPorts_devel_chai/chai/work/CHAI-2024.02.1/tests/integration/raja-chai-launch.cpp:164:33: error: no matching function for call to 'loop<<expression error> >(RAJA::LaunchContext&, RAJA::RangeSegment, cuda_test_Chai_LaunchView()::<lambda(RAJA::LaunchContext)>::<lambda(int)>)'
  164 |        RAJA::loop<LOOP_POLICY_1>(ctx, RAJA::RangeSegment(0, X), [&] (int i) {
      |        ~~~~~~~~~~~~~~~~~~~~~~~~~^~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
  165 |            RAJA::loop<LOOP_POLICY_0>(ctx, RAJA::RangeSegment(0, Y), [&] (int j) {
      |            ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
  166 |                v1(i, j) = (i + (j * X)) * 1.0f;
      |                ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
  167 |              });
      |              ~~~                 
  168 |          });
      |          ~~                      
/opt/local/include/RAJA/pattern/launch/launch_core.hpp:546:35: note: candidate: 'template<class POLICY_LIST, class CONTEXT, class SEGMENT, class BODY> void RAJA::loop(const CONTEXT&, const SEGMENT&, const BODY&)'
  546 | RAJA_HOST_DEVICE RAJA_INLINE void loop(CONTEXT const &ctx,
      |                                   ^~~~
/opt/local/include/RAJA/pattern/launch/launch_core.hpp:546:35: note:   template argument deduction/substitution failed:
/opt/local/var/macports/build/_opt_PPCSnowLeopardPorts_devel_chai/chai/work/CHAI-2024.02.1/tests/integration/raja-chai-launch.cpp:164:33: error: template argument 1 is invalid
  164 |        RAJA::loop<LOOP_POLICY_1>(ctx, RAJA::RangeSegment(0, X), [&] (int i) {
      |        ~~~~~~~~~~~~~~~~~~~~~~~~~^~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
  165 |            RAJA::loop<LOOP_POLICY_0>(ctx, RAJA::RangeSegment(0, Y), [&] (int j) {
      |            ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
  166 |                v1(i, j) = (i + (j * X)) * 1.0f;
      |                ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
  167 |              });
      |              ~~~                 
  168 |          });
      |          ~~                      
/opt/local/var/macports/build/_opt_PPCSnowLeopardPorts_devel_chai/chai/work/CHAI-2024.02.1/tests/integration/raja-chai-launch.cpp: In lambda function:
/opt/local/var/macports/build/_opt_PPCSnowLeopardPorts_devel_chai/chai/work/CHAI-2024.02.1/tests/integration/raja-chai-launch.cpp:177:23: error: 'PARALLEL_LOOP_POLICY_0' was not declared in this scope; did you mean 'PARALLEL_LOOP_POLICY_1'?
  177 |            RAJA::loop<PARALLEL_LOOP_POLICY_0>(ctx, RAJA::RangeSegment(0, Y), [&] (int j) {
      |                       ^~~~~~~~~~~~~~~~~~~~~~
      |                       PARALLEL_LOOP_POLICY_1
/opt/local/var/macports/build/_opt_PPCSnowLeopardPorts_devel_chai/chai/work/CHAI-2024.02.1/tests/integration/raja-chai-launch.cpp:177:46: error: no matching function for call to 'loop<<expression error> >(RAJA::LaunchContext&, RAJA::RangeSegment, cuda_test_Chai_LaunchView()::<lambda(RAJA::LaunchContext)>::<lambda(int)>::<lambda(int)>)'
  177 |            RAJA::loop<PARALLEL_LOOP_POLICY_0>(ctx, RAJA::RangeSegment(0, Y), [&] (int j) {
      |            ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~^~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
  178 |                v2(i, j) = v1(i, j) * 2.0f;
      |                ~~~~~~~~~~~~~~~~~~~~~~~~~~~    
  179 |              });
      |              ~~                               
/opt/local/include/RAJA/pattern/launch/launch_core.hpp:546:35: note: candidate: 'template<class POLICY_LIST, class CONTEXT, class SEGMENT, class BODY> void RAJA::loop(const CONTEXT&, const SEGMENT&, const BODY&)'
  546 | RAJA_HOST_DEVICE RAJA_INLINE void loop(CONTEXT const &ctx,
      |                                   ^~~~
/opt/local/include/RAJA/pattern/launch/launch_core.hpp:546:35: note:   template argument deduction/substitution failed:
/opt/local/var/macports/build/_opt_PPCSnowLeopardPorts_devel_chai/chai/work/CHAI-2024.02.1/tests/integration/raja-chai-launch.cpp:177:46: error: template argument 1 is invalid
  177 |            RAJA::loop<PARALLEL_LOOP_POLICY_0>(ctx, RAJA::RangeSegment(0, Y), [&] (int j) {
      |            ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~^~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
  178 |                v2(i, j) = v1(i, j) * 2.0f;
      |                ~~~~~~~~~~~~~~~~~~~~~~~~~~~    
  179 |              });
      |              ~~                               
/opt/local/var/macports/build/_opt_PPCSnowLeopardPorts_devel_chai/chai/work/CHAI-2024.02.1/tests/integration/raja-chai-launch.cpp: In lambda function:
/opt/local/var/macports/build/_opt_PPCSnowLeopardPorts_devel_chai/chai/work/CHAI-2024.02.1/tests/integration/raja-chai-launch.cpp:188:19: error: 'LOOP_POLICY_1' was not declared in this scope
  188 |        RAJA::loop<LOOP_POLICY_1>(ctx, RAJA::RangeSegment(0, X), [&] (int i) {
      |                   ^~~~~~~~~~~~~
/opt/local/var/macports/build/_opt_PPCSnowLeopardPorts_devel_chai/chai/work/CHAI-2024.02.1/tests/integration/raja-chai-launch.cpp: In lambda function:
/opt/local/var/macports/build/_opt_PPCSnowLeopardPorts_devel_chai/chai/work/CHAI-2024.02.1/tests/integration/raja-chai-launch.cpp:189:23: error: 'LOOP_POLICY_0' was not declared in this scope
  189 |            RAJA::loop<LOOP_POLICY_0>(ctx, RAJA::RangeSegment(0, Y), [&] (int j) {
      |                       ^~~~~~~~~~~~~
/opt/local/var/macports/build/_opt_PPCSnowLeopardPorts_devel_chai/chai/work/CHAI-2024.02.1/tests/integration/raja-chai-launch.cpp:189:37: error: no matching function for call to 'loop<<expression error> >(RAJA::LaunchContext&, RAJA::RangeSegment, cuda_test_Chai_LaunchView()::<lambda(RAJA::LaunchContext)>::<lambda(int)>::<lambda(int)>)'
  189 |            RAJA::loop<LOOP_POLICY_0>(ctx, RAJA::RangeSegment(0, Y), [&] (int j) {
      |            ~~~~~~~~~~~~~~~~~~~~~~~~~^~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
  190 |                ASSERT_FLOAT_EQ(v2(i, j), v1(i, j) * 2.0f);
      |                ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
  191 |              });
      |              ~~                      
/opt/local/include/RAJA/pattern/launch/launch_core.hpp:546:35: note: candidate: 'template<class POLICY_LIST, class CONTEXT, class SEGMENT, class BODY> void RAJA::loop(const CONTEXT&, const SEGMENT&, const BODY&)'
  546 | RAJA_HOST_DEVICE RAJA_INLINE void loop(CONTEXT const &ctx,
      |                                   ^~~~
/opt/local/include/RAJA/pattern/launch/launch_core.hpp:546:35: note:   template argument deduction/substitution failed:
/opt/local/var/macports/build/_opt_PPCSnowLeopardPorts_devel_chai/chai/work/CHAI-2024.02.1/tests/integration/raja-chai-launch.cpp:189:37: error: template argument 1 is invalid
  189 |            RAJA::loop<LOOP_POLICY_0>(ctx, RAJA::RangeSegment(0, Y), [&] (int j) {
      |            ~~~~~~~~~~~~~~~~~~~~~~~~~^~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
  190 |                ASSERT_FLOAT_EQ(v2(i, j), v1(i, j) * 2.0f);
      |                ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
  191 |              });
      |              ~~                      
/opt/local/var/macports/build/_opt_PPCSnowLeopardPorts_devel_chai/chai/work/CHAI-2024.02.1/tests/integration/raja-chai-launch.cpp: In lambda function:
/opt/local/var/macports/build/_opt_PPCSnowLeopardPorts_devel_chai/chai/work/CHAI-2024.02.1/tests/integration/raja-chai-launch.cpp:188:33: error: no matching function for call to 'loop<<expression error> >(RAJA::LaunchContext&, RAJA::RangeSegment, cuda_test_Chai_LaunchView()::<lambda(RAJA::LaunchContext)>::<lambda(int)>)'
  188 |        RAJA::loop<LOOP_POLICY_1>(ctx, RAJA::RangeSegment(0, X), [&] (int i) {
      |        ~~~~~~~~~~~~~~~~~~~~~~~~~^~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
  189 |            RAJA::loop<LOOP_POLICY_0>(ctx, RAJA::RangeSegment(0, Y), [&] (int j) {
      |            ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
  190 |                ASSERT_FLOAT_EQ(v2(i, j), v1(i, j) * 2.0f);
      |                ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
  191 |              });
      |              ~~~                 
  192 |          });
      |          ~~                      
/opt/local/include/RAJA/pattern/launch/launch_core.hpp:546:35: note: candidate: 'template<class POLICY_LIST, class CONTEXT, class SEGMENT, class BODY> void RAJA::loop(const CONTEXT&, const SEGMENT&, const BODY&)'
  546 | RAJA_HOST_DEVICE RAJA_INLINE void loop(CONTEXT const &ctx,
      |                                   ^~~~
/opt/local/include/RAJA/pattern/launch/launch_core.hpp:546:35: note:   template argument deduction/substitution failed:
/opt/local/var/macports/build/_opt_PPCSnowLeopardPorts_devel_chai/chai/work/CHAI-2024.02.1/tests/integration/raja-chai-launch.cpp:188:33: error: template argument 1 is invalid
  188 |        RAJA::loop<LOOP_POLICY_1>(ctx, RAJA::RangeSegment(0, X), [&] (int i) {
      |        ~~~~~~~~~~~~~~~~~~~~~~~~~^~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
  189 |            RAJA::loop<LOOP_POLICY_0>(ctx, RAJA::RangeSegment(0, Y), [&] (int j) {
      |            ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
  190 |                ASSERT_FLOAT_EQ(v2(i, j), v1(i, j) * 2.0f);
      |                ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
  191 |              });
      |              ~~~                 
  192 |          });
      |          ~~                      
/opt/local/var/macports/build/_opt_PPCSnowLeopardPorts_devel_chai/chai/work/CHAI-2024.02.1/tests/integration/raja-chai-launch.cpp: In function 'void cuda_test_Chai_LaunchMultiView()':
/opt/local/var/macports/build/_opt_PPCSnowLeopardPorts_devel_chai/chai/work/CHAI-2024.02.1/tests/integration/raja-chai-launch.cpp:203:48: error: 'loop_exec' is not a member of 'RAJA'
  203 |   using LOOP_POLICY_0 = RAJA::LoopPolicy<RAJA::loop_exec>;
      |                                                ^~~~~~~~~
/opt/local/var/macports/build/_opt_PPCSnowLeopardPorts_devel_chai/chai/work/CHAI-2024.02.1/tests/integration/raja-chai-launch.cpp:203:57: error: template argument 1 is invalid
  203 |   using LOOP_POLICY_0 = RAJA::LoopPolicy<RAJA::loop_exec>;
      |                                                         ^
/opt/local/var/macports/build/_opt_PPCSnowLeopardPorts_devel_chai/chai/work/CHAI-2024.02.1/tests/integration/raja-chai-launch.cpp:204:48: error: 'loop_exec' is not a member of 'RAJA'
  204 |   using LOOP_POLICY_1 = RAJA::LoopPolicy<RAJA::loop_exec>;
      |                                                ^~~~~~~~~
/opt/local/var/macports/build/_opt_PPCSnowLeopardPorts_devel_chai/chai/work/CHAI-2024.02.1/tests/integration/raja-chai-launch.cpp:204:57: error: template argument 1 is invalid
  204 |   using LOOP_POLICY_1 = RAJA::LoopPolicy<RAJA::loop_exec>;
      |                                                         ^
/opt/local/var/macports/build/_opt_PPCSnowLeopardPorts_devel_chai/chai/work/CHAI-2024.02.1/tests/integration/raja-chai-launch.cpp:216:57: error: 'loop_exec' is not a member of 'RAJA'
  216 |   using PARALLEL_LOOP_POLICY_0 = RAJA::LoopPolicy<RAJA::loop_exec>;
      |                                                         ^~~~~~~~~
/opt/local/var/macports/build/_opt_PPCSnowLeopardPorts_devel_chai/chai/work/CHAI-2024.02.1/tests/integration/raja-chai-launch.cpp:216:66: error: template argument 1 is invalid
  216 |   using PARALLEL_LOOP_POLICY_0 = RAJA::LoopPolicy<RAJA::loop_exec>;
      |                                                                  ^
/opt/local/var/macports/build/_opt_PPCSnowLeopardPorts_devel_chai/chai/work/CHAI-2024.02.1/tests/integration/raja-chai-launch.cpp: In lambda function:
/opt/local/var/macports/build/_opt_PPCSnowLeopardPorts_devel_chai/chai/work/CHAI-2024.02.1/tests/integration/raja-chai-launch.cpp:248:19: error: 'LOOP_POLICY_1' was not declared in this scope
  248 |        RAJA::loop<LOOP_POLICY_1>(ctx, RAJA::RangeSegment(0, X), [&] (int i) {
      |                   ^~~~~~~~~~~~~
/opt/local/var/macports/build/_opt_PPCSnowLeopardPorts_devel_chai/chai/work/CHAI-2024.02.1/tests/integration/raja-chai-launch.cpp: In lambda function:
/opt/local/var/macports/build/_opt_PPCSnowLeopardPorts_devel_chai/chai/work/CHAI-2024.02.1/tests/integration/raja-chai-launch.cpp:249:23: error: 'LOOP_POLICY_0' was not declared in this scope
  249 |            RAJA::loop<LOOP_POLICY_0>(ctx, RAJA::RangeSegment(0, Y), [&] (int j) {
      |                       ^~~~~~~~~~~~~
/opt/local/var/macports/build/_opt_PPCSnowLeopardPorts_devel_chai/chai/work/CHAI-2024.02.1/tests/integration/raja-chai-launch.cpp:249:37: error: no matching function for call to 'loop<<expression error> >(RAJA::LaunchContext&, RAJA::RangeSegment, cuda_test_Chai_LaunchMultiView()::<lambda(RAJA::LaunchContext)>::<lambda(int)>::<lambda(int)>)'
  249 |            RAJA::loop<LOOP_POLICY_0>(ctx, RAJA::RangeSegment(0, Y), [&] (int j) {
      |            ~~~~~~~~~~~~~~~~~~~~~~~~~^~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
  250 |                mview(0, i, j) = (i + (j * X)) * 1.0f;
      |                ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
  251 |              });
      |              ~~                      
/opt/local/include/RAJA/pattern/launch/launch_core.hpp:546:35: note: candidate: 'template<class POLICY_LIST, class CONTEXT, class SEGMENT, class BODY> void RAJA::loop(const CONTEXT&, const SEGMENT&, const BODY&)'
  546 | RAJA_HOST_DEVICE RAJA_INLINE void loop(CONTEXT const &ctx,
      |                                   ^~~~
/opt/local/include/RAJA/pattern/launch/launch_core.hpp:546:35: note:   template argument deduction/substitution failed:
/opt/local/var/macports/build/_opt_PPCSnowLeopardPorts_devel_chai/chai/work/CHAI-2024.02.1/tests/integration/raja-chai-launch.cpp:249:37: error: template argument 1 is invalid
  249 |            RAJA::loop<LOOP_POLICY_0>(ctx, RAJA::RangeSegment(0, Y), [&] (int j) {
      |            ~~~~~~~~~~~~~~~~~~~~~~~~~^~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
  250 |                mview(0, i, j) = (i + (j * X)) * 1.0f;
      |                ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
  251 |              });
      |              ~~                      
/opt/local/var/macports/build/_opt_PPCSnowLeopardPorts_devel_chai/chai/work/CHAI-2024.02.1/tests/integration/raja-chai-launch.cpp: In lambda function:
/opt/local/var/macports/build/_opt_PPCSnowLeopardPorts_devel_chai/chai/work/CHAI-2024.02.1/tests/integration/raja-chai-launch.cpp:248:33: error: no matching function for call to 'loop<<expression error> >(RAJA::LaunchContext&, RAJA::RangeSegment, cuda_test_Chai_LaunchMultiView()::<lambda(RAJA::LaunchContext)>::<lambda(int)>)'
  248 |        RAJA::loop<LOOP_POLICY_1>(ctx, RAJA::RangeSegment(0, X), [&] (int i) {
      |        ~~~~~~~~~~~~~~~~~~~~~~~~~^~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
  249 |            RAJA::loop<LOOP_POLICY_0>(ctx, RAJA::RangeSegment(0, Y), [&] (int j) {
      |            ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
  250 |                mview(0, i, j) = (i + (j * X)) * 1.0f;
      |                ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
  251 |              });
      |              ~~~                 
  252 |          });
      |          ~~                      
/opt/local/include/RAJA/pattern/launch/launch_core.hpp:546:35: note: candidate: 'template<class POLICY_LIST, class CONTEXT, class SEGMENT, class BODY> void RAJA::loop(const CONTEXT&, const SEGMENT&, const BODY&)'
  546 | RAJA_HOST_DEVICE RAJA_INLINE void loop(CONTEXT const &ctx,
      |                                   ^~~~
/opt/local/include/RAJA/pattern/launch/launch_core.hpp:546:35: note:   template argument deduction/substitution failed:
/opt/local/var/macports/build/_opt_PPCSnowLeopardPorts_devel_chai/chai/work/CHAI-2024.02.1/tests/integration/raja-chai-launch.cpp:248:33: error: template argument 1 is invalid
  248 |        RAJA::loop<LOOP_POLICY_1>(ctx, RAJA::RangeSegment(0, X), [&] (int i) {
      |        ~~~~~~~~~~~~~~~~~~~~~~~~~^~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
  249 |            RAJA::loop<LOOP_POLICY_0>(ctx, RAJA::RangeSegment(0, Y), [&] (int j) {
      |            ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
  250 |                mview(0, i, j) = (i + (j * X)) * 1.0f;
      |                ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
  251 |              });
      |              ~~~                 
  252 |          });
      |          ~~                      
/opt/local/var/macports/build/_opt_PPCSnowLeopardPorts_devel_chai/chai/work/CHAI-2024.02.1/tests/integration/raja-chai-launch.cpp: In lambda function:
/opt/local/var/macports/build/_opt_PPCSnowLeopardPorts_devel_chai/chai/work/CHAI-2024.02.1/tests/integration/raja-chai-launch.cpp:261:23: error: 'PARALLEL_LOOP_POLICY_0' was not declared in this scope; did you mean 'PARALLEL_LOOP_POLICY_1'?
  261 |            RAJA::loop<PARALLEL_LOOP_POLICY_0>(ctx, RAJA::RangeSegment(0, Y), [&] (int j) {
      |                       ^~~~~~~~~~~~~~~~~~~~~~
      |                       PARALLEL_LOOP_POLICY_1
/opt/local/var/macports/build/_opt_PPCSnowLeopardPorts_devel_chai/chai/work/CHAI-2024.02.1/tests/integration/raja-chai-launch.cpp:261:46: error: no matching function for call to 'loop<<expression error> >(RAJA::LaunchContext&, RAJA::RangeSegment, cuda_test_Chai_LaunchMultiView()::<lambda(RAJA::LaunchContext)>::<lambda(int)>::<lambda(int)>)'
  261 |            RAJA::loop<PARALLEL_LOOP_POLICY_0>(ctx, RAJA::RangeSegment(0, Y), [&] (int j) {
      |            ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~^~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
  262 |                // use both MultiViews
      |                ~~~~~~~~~~~~~~~~~~~~~~         
  263 |                mview(1, i, j) = mview1p(i, 0, j) * 2.0f;
      |                ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
  264 |              });
      |              ~~                               
/opt/local/include/RAJA/pattern/launch/launch_core.hpp:546:35: note: candidate: 'template<class POLICY_LIST, class CONTEXT, class SEGMENT, class BODY> void RAJA::loop(const CONTEXT&, const SEGMENT&, const BODY&)'
  546 | RAJA_HOST_DEVICE RAJA_INLINE void loop(CONTEXT const &ctx,
      |                                   ^~~~
/opt/local/include/RAJA/pattern/launch/launch_core.hpp:546:35: note:   template argument deduction/substitution failed:
/opt/local/var/macports/build/_opt_PPCSnowLeopardPorts_devel_chai/chai/work/CHAI-2024.02.1/tests/integration/raja-chai-launch.cpp:261:46: error: template argument 1 is invalid
  261 |            RAJA::loop<PARALLEL_LOOP_POLICY_0>(ctx, RAJA::RangeSegment(0, Y), [&] (int j) {
      |            ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~^~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
  262 |                // use both MultiViews
      |                ~~~~~~~~~~~~~~~~~~~~~~         
  263 |                mview(1, i, j) = mview1p(i, 0, j) * 2.0f;
      |                ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
  264 |              });
      |              ~~                               
/opt/local/var/macports/build/_opt_PPCSnowLeopardPorts_devel_chai/chai/work/CHAI-2024.02.1/tests/integration/raja-chai-launch.cpp: In lambda function:
/opt/local/var/macports/build/_opt_PPCSnowLeopardPorts_devel_chai/chai/work/CHAI-2024.02.1/tests/integration/raja-chai-launch.cpp:272:19: error: 'LOOP_POLICY_1' was not declared in this scope
  272 |        RAJA::loop<LOOP_POLICY_1>(ctx, RAJA::RangeSegment(0, X), [&] (int i) {
      |                   ^~~~~~~~~~~~~
/opt/local/var/macports/build/_opt_PPCSnowLeopardPorts_devel_chai/chai/work/CHAI-2024.02.1/tests/integration/raja-chai-launch.cpp: In lambda function:
/opt/local/var/macports/build/_opt_PPCSnowLeopardPorts_devel_chai/chai/work/CHAI-2024.02.1/tests/integration/raja-chai-launch.cpp:273:23: error: 'LOOP_POLICY_0' was not declared in this scope
  273 |            RAJA::loop<LOOP_POLICY_0>(ctx, RAJA::RangeSegment(0, Y), [&] (int j) {
      |                       ^~~~~~~~~~~~~
/opt/local/var/macports/build/_opt_PPCSnowLeopardPorts_devel_chai/chai/work/CHAI-2024.02.1/tests/integration/raja-chai-launch.cpp:273:37: error: no matching function for call to 'loop<<expression error> >(RAJA::LaunchContext&, RAJA::RangeSegment, cuda_test_Chai_LaunchMultiView()::<lambda(RAJA::LaunchContext)>::<lambda(int)>::<lambda(int)>)'
  273 |            RAJA::loop<LOOP_POLICY_0>(ctx, RAJA::RangeSegment(0, Y), [&] (int j) {
      |            ~~~~~~~~~~~~~~~~~~~~~~~~~^~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
  274 |                ASSERT_FLOAT_EQ(mview(1, i, j), mview(0, i, j) * 2.0f);
      |                ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
  275 |              });
      |              ~~                      
/opt/local/include/RAJA/pattern/launch/launch_core.hpp:546:35: note: candidate: 'template<class POLICY_LIST, class CONTEXT, class SEGMENT, class BODY> void RAJA::loop(const CONTEXT&, const SEGMENT&, const BODY&)'
  546 | RAJA_HOST_DEVICE RAJA_INLINE void loop(CONTEXT const &ctx,
      |                                   ^~~~
/opt/local/include/RAJA/pattern/launch/launch_core.hpp:546:35: note:   template argument deduction/substitution failed:
/opt/local/var/macports/build/_opt_PPCSnowLeopardPorts_devel_chai/chai/work/CHAI-2024.02.1/tests/integration/raja-chai-launch.cpp:273:37: error: template argument 1 is invalid
  273 |            RAJA::loop<LOOP_POLICY_0>(ctx, RAJA::RangeSegment(0, Y), [&] (int j) {
      |            ~~~~~~~~~~~~~~~~~~~~~~~~~^~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
  274 |                ASSERT_FLOAT_EQ(mview(1, i, j), mview(0, i, j) * 2.0f);
      |                ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
  275 |              });
      |              ~~                      
/opt/local/var/macports/build/_opt_PPCSnowLeopardPorts_devel_chai/chai/work/CHAI-2024.02.1/tests/integration/raja-chai-launch.cpp: In lambda function:
/opt/local/var/macports/build/_opt_PPCSnowLeopardPorts_devel_chai/chai/work/CHAI-2024.02.1/tests/integration/raja-chai-launch.cpp:272:33: error: no matching function for call to 'loop<<expression error> >(RAJA::LaunchContext&, RAJA::RangeSegment, cuda_test_Chai_LaunchMultiView()::<lambda(RAJA::LaunchContext)>::<lambda(int)>)'
  272 |        RAJA::loop<LOOP_POLICY_1>(ctx, RAJA::RangeSegment(0, X), [&] (int i) {
      |        ~~~~~~~~~~~~~~~~~~~~~~~~~^~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
  273 |            RAJA::loop<LOOP_POLICY_0>(ctx, RAJA::RangeSegment(0, Y), [&] (int j) {
      |            ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
  274 |                ASSERT_FLOAT_EQ(mview(1, i, j), mview(0, i, j) * 2.0f);
      |                ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
  275 |              });
      |              ~~~                 
  276 |          });
      |          ~~                      
/opt/local/include/RAJA/pattern/launch/launch_core.hpp:546:35: note: candidate: 'template<class POLICY_LIST, class CONTEXT, class SEGMENT, class BODY> void RAJA::loop(const CONTEXT&, const SEGMENT&, const BODY&)'
  546 | RAJA_HOST_DEVICE RAJA_INLINE void loop(CONTEXT const &ctx,
      |                                   ^~~~
/opt/local/include/RAJA/pattern/launch/launch_core.hpp:546:35: note:   template argument deduction/substitution failed:
/opt/local/var/macports/build/_opt_PPCSnowLeopardPorts_devel_chai/chai/work/CHAI-2024.02.1/tests/integration/raja-chai-launch.cpp:272:33: error: template argument 1 is invalid
  272 |        RAJA::loop<LOOP_POLICY_1>(ctx, RAJA::RangeSegment(0, X), [&] (int i) {
      |        ~~~~~~~~~~~~~~~~~~~~~~~~~^~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
  273 |            RAJA::loop<LOOP_POLICY_0>(ctx, RAJA::RangeSegment(0, Y), [&] (int j) {
      |            ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
  274 |                ASSERT_FLOAT_EQ(mview(1, i, j), mview(0, i, j) * 2.0f);
      |                ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
  275 |              });
      |              ~~~                 
  276 |          });
      |          ~~                      
/opt/local/var/macports/build/_opt_PPCSnowLeopardPorts_devel_chai/chai/work/CHAI-2024.02.1/tests/integration/raja-chai-launch.cpp: In function 'void runLTimesTests(RAJA::Index_type, RAJA::Index_type, RAJA::Index_type, RAJA::Index_type)':
/opt/local/var/macports/build/_opt_PPCSnowLeopardPorts_devel_chai/chai/work/CHAI-2024.02.1/tests/integration/raja-chai-launch.cpp:355:57: error: 'loop_exec' is not a member of 'RAJA'
  355 |   using PARALLEL_LOOP_POLICY_0 = RAJA::LoopPolicy<RAJA::loop_exec>;
      |                                                         ^~~~~~~~~
/opt/local/var/macports/build/_opt_PPCSnowLeopardPorts_devel_chai/chai/work/CHAI-2024.02.1/tests/integration/raja-chai-launch.cpp:355:66: error: template argument 1 is invalid
  355 |   using PARALLEL_LOOP_POLICY_0 = RAJA::LoopPolicy<RAJA::loop_exec>;
      |                                                                  ^
/opt/local/var/macports/build/_opt_PPCSnowLeopardPorts_devel_chai/chai/work/CHAI-2024.02.1/tests/integration/raja-chai-launch.cpp:356:57: error: 'loop_exec' is not a member of 'RAJA'
  356 |   using PARALLEL_LOOP_POLICY_1 = RAJA::LoopPolicy<RAJA::loop_exec>;
      |                                                         ^~~~~~~~~
/opt/local/var/macports/build/_opt_PPCSnowLeopardPorts_devel_chai/chai/work/CHAI-2024.02.1/tests/integration/raja-chai-launch.cpp:356:66: error: template argument 1 is invalid
  356 |   using PARALLEL_LOOP_POLICY_1 = RAJA::LoopPolicy<RAJA::loop_exec>;
      |                                                                  ^
/opt/local/var/macports/build/_opt_PPCSnowLeopardPorts_devel_chai/chai/work/CHAI-2024.02.1/tests/integration/raja-chai-launch.cpp:357:57: error: 'loop_exec' is not a member of 'RAJA'
  357 |   using PARALLEL_LOOP_POLICY_2 = RAJA::LoopPolicy<RAJA::loop_exec>;
      |                                                         ^~~~~~~~~
/opt/local/var/macports/build/_opt_PPCSnowLeopardPorts_devel_chai/chai/work/CHAI-2024.02.1/tests/integration/raja-chai-launch.cpp:357:66: error: template argument 1 is invalid
  357 |   using PARALLEL_LOOP_POLICY_2 = RAJA::LoopPolicy<RAJA::loop_exec>;
      |                                                                  ^
/opt/local/var/macports/build/_opt_PPCSnowLeopardPorts_devel_chai/chai/work/CHAI-2024.02.1/tests/integration/raja-chai-launch.cpp: In lambda function:
/opt/local/var/macports/build/_opt_PPCSnowLeopardPorts_devel_chai/chai/work/CHAI-2024.02.1/tests/integration/raja-chai-launch.cpp:372:23: error: 'PARALLEL_LOOP_POLICY_2' was not declared in this scope; did you mean 'PARALLEL_LOOP_POLICY_3'?
  372 |            RAJA::loop<PARALLEL_LOOP_POLICY_2>(ctx, RAJA::TypedRangeSegment<IG>(0, num_groups), [&] (IG g) {
      |                       ^~~~~~~~~~~~~~~~~~~~~~
      |                       PARALLEL_LOOP_POLICY_3
/opt/local/var/macports/build/_opt_PPCSnowLeopardPorts_devel_chai/chai/work/CHAI-2024.02.1/tests/integration/raja-chai-launch.cpp: In lambda function:
/opt/local/var/macports/build/_opt_PPCSnowLeopardPorts_devel_chai/chai/work/CHAI-2024.02.1/tests/integration/raja-chai-launch.cpp:373:27: error: 'PARALLEL_LOOP_POLICY_1' was not declared in this scope; did you mean 'PARALLEL_LOOP_POLICY_3'?
  373 |                RAJA::loop<PARALLEL_LOOP_POLICY_1>(ctx, RAJA::TypedRangeSegment<IZ>(0, num_zones ), [&] (IZ z) {
      |                           ^~~~~~~~~~~~~~~~~~~~~~
      |                           PARALLEL_LOOP_POLICY_3
/opt/local/var/macports/build/_opt_PPCSnowLeopardPorts_devel_chai/chai/work/CHAI-2024.02.1/tests/integration/raja-chai-launch.cpp: In lambda function:
/opt/local/var/macports/build/_opt_PPCSnowLeopardPorts_devel_chai/chai/work/CHAI-2024.02.1/tests/integration/raja-chai-launch.cpp:374:31: error: 'PARALLEL_LOOP_POLICY_0' was not declared in this scope; did you mean 'PARALLEL_LOOP_POLICY_3'?
  374 |                    RAJA::loop<PARALLEL_LOOP_POLICY_0>(ctx, RAJA::TypedRangeSegment<ID>(0, num_directions ), [&] (ID d) {
      |                               ^~~~~~~~~~~~~~~~~~~~~~
      |                               PARALLEL_LOOP_POLICY_3
/opt/local/var/macports/build/_opt_PPCSnowLeopardPorts_devel_chai/chai/work/CHAI-2024.02.1/tests/integration/raja-chai-launch.cpp:374:54: error: no matching function for call to 'loop<<expression error> >(RAJA::LaunchContext&, RAJA::TypedRangeSegment<ID>, runLTimesTests(RAJA::Index_type, RAJA::Index_type, RAJA::Index_type, RAJA::Index_type)::<lambda(RAJA::LaunchContext)>::<lambda(IM)>::<lambda(IG)>::<lambda(IZ)>::<lambda(ID)>)'
  374 |                    RAJA::loop<PARALLEL_LOOP_POLICY_0>(ctx, RAJA::TypedRangeSegment<ID>(0, num_directions ), [&] (ID d) {
      |                    ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~^~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
  375 |                        phi(m, g, z) += L(m, d) * psi(d, g, z);
      |                        ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
  376 |                      });
      |                      ~~                               
/opt/local/include/RAJA/pattern/launch/launch_core.hpp:546:35: note: candidate: 'template<class POLICY_LIST, class CONTEXT, class SEGMENT, class BODY> void RAJA::loop(const CONTEXT&, const SEGMENT&, const BODY&)'
  546 | RAJA_HOST_DEVICE RAJA_INLINE void loop(CONTEXT const &ctx,
      |                                   ^~~~
/opt/local/include/RAJA/pattern/launch/launch_core.hpp:546:35: note:   template argument deduction/substitution failed:
/opt/local/var/macports/build/_opt_PPCSnowLeopardPorts_devel_chai/chai/work/CHAI-2024.02.1/tests/integration/raja-chai-launch.cpp:374:54: error: template argument 1 is invalid
  374 |                    RAJA::loop<PARALLEL_LOOP_POLICY_0>(ctx, RAJA::TypedRangeSegment<ID>(0, num_directions ), [&] (ID d) {
      |                    ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~^~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
  375 |                        phi(m, g, z) += L(m, d) * psi(d, g, z);
      |                        ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
  376 |                      });
      |                      ~~                               
/opt/local/var/macports/build/_opt_PPCSnowLeopardPorts_devel_chai/chai/work/CHAI-2024.02.1/tests/integration/raja-chai-launch.cpp: In lambda function:
/opt/local/var/macports/build/_opt_PPCSnowLeopardPorts_devel_chai/chai/work/CHAI-2024.02.1/tests/integration/raja-chai-launch.cpp:373:50: error: no matching function for call to 'loop<<expression error> >(RAJA::LaunchContext&, RAJA::TypedRangeSegment<IZ>, runLTimesTests(RAJA::Index_type, RAJA::Index_type, RAJA::Index_type, RAJA::Index_type)::<lambda(RAJA::LaunchContext)>::<lambda(IM)>::<lambda(IG)>::<lambda(IZ)>)'
  373 |                RAJA::loop<PARALLEL_LOOP_POLICY_1>(ctx, RAJA::TypedRangeSegment<IZ>(0, num_zones ), [&] (IZ z) {
      |                ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~^~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
  374 |                    RAJA::loop<PARALLEL_LOOP_POLICY_0>(ctx, RAJA::TypedRangeSegment<ID>(0, num_directions ), [&] (ID d) {
      |                    ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
  375 |                        phi(m, g, z) += L(m, d) * psi(d, g, z);
      |                        ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
  376 |                      });
      |                      ~~~                          
  377 |                  });
      |                  ~~                               
/opt/local/include/RAJA/pattern/launch/launch_core.hpp:546:35: note: candidate: 'template<class POLICY_LIST, class CONTEXT, class SEGMENT, class BODY> void RAJA::loop(const CONTEXT&, const SEGMENT&, const BODY&)'
  546 | RAJA_HOST_DEVICE RAJA_INLINE void loop(CONTEXT const &ctx,
      |                                   ^~~~
/opt/local/include/RAJA/pattern/launch/launch_core.hpp:546:35: note:   template argument deduction/substitution failed:
/opt/local/var/macports/build/_opt_PPCSnowLeopardPorts_devel_chai/chai/work/CHAI-2024.02.1/tests/integration/raja-chai-launch.cpp:373:50: error: template argument 1 is invalid
  373 |                RAJA::loop<PARALLEL_LOOP_POLICY_1>(ctx, RAJA::TypedRangeSegment<IZ>(0, num_zones ), [&] (IZ z) {
      |                ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~^~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
  374 |                    RAJA::loop<PARALLEL_LOOP_POLICY_0>(ctx, RAJA::TypedRangeSegment<ID>(0, num_directions ), [&] (ID d) {
      |                    ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
  375 |                        phi(m, g, z) += L(m, d) * psi(d, g, z);
      |                        ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
  376 |                      });
      |                      ~~~                          
  377 |                  });
      |                  ~~                               
/opt/local/var/macports/build/_opt_PPCSnowLeopardPorts_devel_chai/chai/work/CHAI-2024.02.1/tests/integration/raja-chai-launch.cpp: In lambda function:
/opt/local/var/macports/build/_opt_PPCSnowLeopardPorts_devel_chai/chai/work/CHAI-2024.02.1/tests/integration/raja-chai-launch.cpp:372:46: error: no matching function for call to 'loop<<expression error> >(RAJA::LaunchContext&, RAJA::TypedRangeSegment<IG>, runLTimesTests(RAJA::Index_type, RAJA::Index_type, RAJA::Index_type, RAJA::Index_type)::<lambda(RAJA::LaunchContext)>::<lambda(IM)>::<lambda(IG)>)'
  372 |            RAJA::loop<PARALLEL_LOOP_POLICY_2>(ctx, RAJA::TypedRangeSegment<IG>(0, num_groups), [&] (IG g) {
      |            ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~^~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
  373 |                RAJA::loop<PARALLEL_LOOP_POLICY_1>(ctx, RAJA::TypedRangeSegment<IZ>(0, num_zones ), [&] (IZ z) {
      |                ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
  374 |                    RAJA::loop<PARALLEL_LOOP_POLICY_0>(ctx, RAJA::TypedRangeSegment<ID>(0, num_directions ), [&] (ID d) {
      |                    ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
  375 |                        phi(m, g, z) += L(m, d) * psi(d, g, z);
      |                        ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
  376 |                      });
      |                      ~~~                      
  377 |                  });
      |                  ~~~                          
  378 |              });
      |              ~~                               
/opt/local/include/RAJA/pattern/launch/launch_core.hpp:546:35: note: candidate: 'template<class POLICY_LIST, class CONTEXT, class SEGMENT, class BODY> void RAJA::loop(const CONTEXT&, const SEGMENT&, const BODY&)'
  546 | RAJA_HOST_DEVICE RAJA_INLINE void loop(CONTEXT const &ctx,
      |                                   ^~~~
/opt/local/include/RAJA/pattern/launch/launch_core.hpp:546:35: note:   template argument deduction/substitution failed:
/opt/local/var/macports/build/_opt_PPCSnowLeopardPorts_devel_chai/chai/work/CHAI-2024.02.1/tests/integration/raja-chai-launch.cpp:372:46: error: template argument 1 is invalid
  372 |            RAJA::loop<PARALLEL_LOOP_POLICY_2>(ctx, RAJA::TypedRangeSegment<IG>(0, num_groups), [&] (IG g) {
      |            ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~^~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
  373 |                RAJA::loop<PARALLEL_LOOP_POLICY_1>(ctx, RAJA::TypedRangeSegment<IZ>(0, num_zones ), [&] (IZ z) {
      |                ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
  374 |                    RAJA::loop<PARALLEL_LOOP_POLICY_0>(ctx, RAJA::TypedRangeSegment<ID>(0, num_directions ), [&] (ID d) {
      |                    ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
  375 |                        phi(m, g, z) += L(m, d) * psi(d, g, z);
      |                        ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
  376 |                      });
      |                      ~~~                      
  377 |                  });
      |                  ~~~                          
  378 |              });
      |              ~~                               
[ 96%] Linking CXX executable ../../bin/raja-chai-tests
cd /opt/local/var/macports/build/_opt_PPCSnowLeopardPorts_devel_chai/chai/work/build/tests/integration && /opt/local/bin/cmake -E cmake_link_script CMakeFiles/raja-chai-tests.dir/link.txt --verbose=ON
/opt/local/bin/mpicxx-mpich-gcc13  -Wall -Wextra  -pipe -Os -DNDEBUG -isystem/opt/local/include/LegacySupport -I/opt/local/include -D_GLIBCXX_USE_CXX11_ABI=0    -arch ppc -mmacosx-version-min=10.6 -Wl,-search_paths_first -Wl,-headerpad_max_install_names -Wl,-headerpad_max_install_names -L/opt/local/lib -lMacportsLegacySupport -Wl,-rpath,/opt/local/lib/libgcc -rdynamic -fopenmp "CMakeFiles/raja-chai-tests.dir/raja-chai-tests.cpp.o" -o ../../bin/raja-chai-tests  -Wl,-rpath,/opt/local/var/macports/build/_opt_PPCSnowLeopardPorts_devel_chai/chai/work/build/lib ../../lib/libchai.dylib ../../lib/libumpire.dylib ../../lib/libgtest_main.1.13.0.dylib ../../lib/libgtest.1.13.0.dylib /opt/local/lib/libRAJA.a ../../lib/libcamp.dylib
g++-mp-13: note: rdynamic is not supported
make[2]: Leaving directory `/opt/local/var/macports/build/_opt_PPCSnowLeopardPorts_devel_chai/chai/work/build'
[ 96%] Built target raja-chai-tests
make[2]: *** [tests/integration/CMakeFiles/raja-chai-launch-tests.dir/raja-chai-launch.cpp.o] Error 1
make[2]: Leaving directory `/opt/local/var/macports/build/_opt_PPCSnowLeopardPorts_devel_chai/chai/work/build'
make[1]: *** [tests/integration/CMakeFiles/raja-chai-launch-tests.dir/all] Error 2
make[1]: *** Waiting for unfinished jobs....
```

--- a/tests/integration/raja-chai-launch.cpp
+++ b/tests/integration/raja-chai-launch.cpp
@@ -48,8 +48,8 @@ CUDA_TEST(Chai, LaunchSimple)
 {
 
   using LAUNCH_POLICY = RAJA::LaunchPolicy<RAJA::seq_launch_t>;
-  using LOOP_POLICY_0 = RAJA::LoopPolicy<RAJA::loop_exec>;
-  using LOOP_POLICY_1 = RAJA::LoopPolicy<RAJA::loop_exec>;
+  using LOOP_POLICY_0 = RAJA::LoopPolicy<RAJA::seq_exec>;
+  using LOOP_POLICY_1 = RAJA::LoopPolicy<RAJA::seq_exec>;
 
 #if defined(RAJA_ENABLE_CUDA)
 
@@ -61,7 +61,7 @@ CUDA_TEST(Chai, LaunchSimple)
 #elif defined(RAJA_ENABLE_OPENMP)
 
   using PARALLEL_LAUNCH_POLICY = RAJA::LaunchPolicy<RAJA::omp_launch_t>;
-  using PARALLEL_LOOP_POLICY_0 = RAJA::LoopPolicy<RAJA::loop_exec>;
+  using PARALLEL_LOOP_POLICY_0 = RAJA::LoopPolicy<RAJA::seq_exec>;
   using PARALLEL_LOOP_POLICY_1 = RAJA::LoopPolicy<RAJA::omp_for_exec>;
 
 #else
@@ -122,8 +122,8 @@ CUDA_TEST(Chai, LaunchView)
 {
 
   using LAUNCH_POLICY = RAJA::LaunchPolicy<RAJA::seq_launch_t>;
-  using LOOP_POLICY_0 = RAJA::LoopPolicy<RAJA::loop_exec>;
-  using LOOP_POLICY_1 = RAJA::LoopPolicy<RAJA::loop_exec>;
+  using LOOP_POLICY_0 = RAJA::LoopPolicy<RAJA::seq_exec>;
+  using LOOP_POLICY_1 = RAJA::LoopPolicy<RAJA::seq_exec>;
 
 #if defined(RAJA_ENABLE_CUDA)
 
@@ -135,7 +135,7 @@ CUDA_TEST(Chai, LaunchView)
 #elif defined(RAJA_ENABLE_OPENMP)
 
   using PARALLEL_LAUNCH_POLICY = RAJA::LaunchPolicy<RAJA::omp_launch_t>;
-  using PARALLEL_LOOP_POLICY_0 = RAJA::LoopPolicy<RAJA::loop_exec>;
+  using PARALLEL_LOOP_POLICY_0 = RAJA::LoopPolicy<RAJA::seq_exec>;
   using PARALLEL_LOOP_POLICY_1 = RAJA::LoopPolicy<RAJA::omp_for_exec>;
 
 #else
@@ -200,8 +200,8 @@ CUDA_TEST(Chai, LaunchMultiView)
 {
 
   using LAUNCH_POLICY = RAJA::LaunchPolicy<RAJA::seq_launch_t>;
-  using LOOP_POLICY_0 = RAJA::LoopPolicy<RAJA::loop_exec>;
-  using LOOP_POLICY_1 = RAJA::LoopPolicy<RAJA::loop_exec>;
+  using LOOP_POLICY_0 = RAJA::LoopPolicy<RAJA::seq_exec>;
+  using LOOP_POLICY_1 = RAJA::LoopPolicy<RAJA::seq_exec>;
 
 #if defined(RAJA_ENABLE_CUDA)
 
@@ -213,7 +213,7 @@ CUDA_TEST(Chai, LaunchMultiView)
 #elif defined(RAJA_ENABLE_OPENMP)
 
   using PARALLEL_LAUNCH_POLICY = RAJA::LaunchPolicy<RAJA::omp_launch_t>;
-  using PARALLEL_LOOP_POLICY_0 = RAJA::LoopPolicy<RAJA::loop_exec>;
+  using PARALLEL_LOOP_POLICY_0 = RAJA::LoopPolicy<RAJA::seq_exec>;
   using PARALLEL_LOOP_POLICY_1 = RAJA::LoopPolicy<RAJA::omp_for_exec>;
 #else
 
@@ -345,23 +345,23 @@ void runLTimesTests(Index_type num_moments,
 #if defined(RAJA_ENABLE_CUDA)
   const bool async = false;
   using PARALLEL_LAUNCH_POLICY = RAJA::LaunchPolicy<RAJA::cuda_launch_t<async>>;
-  using PARALLEL_LOOP_POLICY_0 = RAJA::LoopPolicy<RAJA::loop_exec>;
+  using PARALLEL_LOOP_POLICY_0 = RAJA::LoopPolicy<RAJA::seq_exec>;
   using PARALLEL_LOOP_POLICY_1 = RAJA::LoopPolicy<RAJA::cuda_thread_x_loop>;
   using PARALLEL_LOOP_POLICY_2 = RAJA::LoopPolicy<RAJA::cuda_block_y_loop>;
   using PARALLEL_LOOP_POLICY_3 = RAJA::LoopPolicy<RAJA::cuda_block_x_loop>;
 
 #elif defined(RAJA_ENABLE_OPENMP)
   using PARALLEL_LAUNCH_POLICY = RAJA::LaunchPolicy<RAJA::omp_launch_t>;
-  using PARALLEL_LOOP_POLICY_0 = RAJA::LoopPolicy<RAJA::loop_exec>;
-  using PARALLEL_LOOP_POLICY_1 = RAJA::LoopPolicy<RAJA::loop_exec>;
-  using PARALLEL_LOOP_POLICY_2 = RAJA::LoopPolicy<RAJA::loop_exec>;
+  using PARALLEL_LOOP_POLICY_0 = RAJA::LoopPolicy<RAJA::seq_exec>;
+  using PARALLEL_LOOP_POLICY_1 = RAJA::LoopPolicy<RAJA::seq_exec>;
+  using PARALLEL_LOOP_POLICY_2 = RAJA::LoopPolicy<RAJA::seq_exec>;
   using PARALLEL_LOOP_POLICY_3 = RAJA::LoopPolicy<RAJA::omp_for_exec>;
 #else
   using PARALLEL_LAUNCH_POLICY = RAJA::LaunchPolicy<RAJA::seq_launch_t>;
-  using PARALLEL_LOOP_POLICY_0 = RAJA::LoopPolicy<RAJA::loop_exec>;
-  using PARALLEL_LOOP_POLICY_1 = RAJA::LoopPolicy<RAJA::loop_exec>;
-  using PARALLEL_LOOP_POLICY_2 = RAJA::LoopPolicy<RAJA::loop_exec>;
-  using PARALLEL_LOOP_POLICY_3 = RAJA::LoopPolicy<RAJA::loop_exec>;
+  using PARALLEL_LOOP_POLICY_0 = RAJA::LoopPolicy<RAJA::seq_exec>;
+  using PARALLEL_LOOP_POLICY_1 = RAJA::LoopPolicy<RAJA::seq_exec>;
+  using PARALLEL_LOOP_POLICY_2 = RAJA::LoopPolicy<RAJA::seq_exec>;
+  using PARALLEL_LOOP_POLICY_3 = RAJA::LoopPolicy<RAJA::seq_exec>;
 #endif
 
     RAJA::launch<PARALLEL_LAUNCH_POLICY>

